doc/connectivity/resting_state_networks.rst

```diff
@@ -91,3 +96,73 @@ Finally, we can plot the map for different ICA components separately:
+
+Beyond ICA : Dictionary learning
+================================
+
+Recent work has shown that dictionary learning based techniques outperform
+ICA in term of stability and constitutes a better first step in a statistical
+analysis pipeline.
+Dictionary learning in neuro-imaging seeks to extract a few representative
+temporal elements along with their sparse spatial loadings, which constitute
+good extracted maps.
+
+.. topic:: **References**
+
+   * Gael Varoquaux et al. `Multi-subject dictionary learning to segment an atlas of brain spontaneous activitym <http://hal.inria.fr/inria-00588898/en/>`_,
+   Information Processing in Medical Imaging, 2011, pp. 562-573, Lecture Notes
+   in Computer Science,
+
+Applying DictLearning
+---------------------
+
+DictLearning is a ready-to-use class with the same interface as CanICA.
+Sparsity of output map is controlled by a parameter alpha: using a
+larger alpha yields sparser maps.
+
+.. literalinclude:: ../../examples/connectivity/plot_compare_resting_state_decomposition.py
+    :start-after: # Dictionary learning
+    :end-before:# CanICA
+
+We can fit both estimators to compare them
+
+.. literalinclude:: ../../examples/connectivity/plot_compare_resting_state_decomposition.py
+    :start-after: # Fitting both estimators
+    :end-before: # Visualize the results
+
+Visualizing the results
+-----------------------
+
+4D plotting offers an efficient way to compare both resulting outputs
+
+.. literalinclude:: ../../examples/connectivity/plot_compare_resting_state_decomposition.py
+    :start-after: # Visualize the results
+
+.. |left_img| image:: ../auto_examples/connectivity/images/plot_compare_resting_state_decomposition_001.png
+   :target: ../auto_examples/plot_compare_resting_state_decomposition.html
+   :width: 50%
+.. |right_img| image:: ../auto_examples/connectivity/images/plot_compare_resting_state_decomposition_003.png
+   :target: ../auto_examples/plot_compare_resting_state_decomposition.html
+   :width: 50%
+
+.. |left_img_single| image:: ../auto_examples/connectivity/images/plot_compare_resting_state_decomposition_002.png
+   :target: ../auto_examples/plot_compare_resting_state_decomposition.html
+   :width: 50%
+.. |right_img_single| image:: ../auto_examples/connectivity/images/plot_compare_resting_state_decomposition_004.png
+   :target: ../auto_examples/plot_compare_resting_state_decomposition.html
+   :width: 50%
+
+
+.. centered:: |left_img| |right_img|
+.. centered:: |left_img_single| |right_img_single|
+
+Maps obtained with dictionary leaning are often easier to exploit as they are
+less noisy than ICA maps, with blobs usually well defined.
+While dictionary learning computation time is comparable to CanICA, obtained
+atlases have been shown to outperform ICA in a variety of
+classification tasks.
+
+.. seealso::
+
+   The full code can be found as an example:
+   :ref:`sphx_glr_auto_examples_plot_compare_resting_state_decomposition.py`
```

Comment:
This reference is wrong. You can grab the right reference from the build of sphinx-gallery, once it is finished.